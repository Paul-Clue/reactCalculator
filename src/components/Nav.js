import React from 'react';
import { Link } from 'react-router-dom';
import '../App.css';

const Nav = () => (
  <div>
    <nav className="navbar navbar-light bg-info">
      <div className="container-fluid">
        <span className="navbar-brand mb-0 h1"><h1>Math Magicians</h1></span>

        <div className="AllLinks">
          <Link className="Link border" to="/">Home</Link>
          <Link className="Link border" to="/calculator">Calculator</Link>
          <Link className="Link" to="/quote">Quote</Link>
        </div>
      </div>
    </nav>
  </div>
);

export default Nav;
