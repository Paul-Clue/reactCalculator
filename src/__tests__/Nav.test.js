import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Nav from '../components/Nav';

describe('Nav Component', () => {
  test('It should render', () => {
    const nav = renderer.create(
      <BrowserRouter>
        <Nav />
      </BrowserRouter>,
    ).toJSON();
    expect(nav).toMatchSnapshot();
  });
});