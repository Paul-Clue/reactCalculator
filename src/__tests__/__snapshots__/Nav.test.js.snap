// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Component It should render 1`] = `
<div>
  <nav
    className="navbar navbar-light bg-info"
  >
    <div
      className="container-fluid"
    >
      <span
        className="navbar-brand mb-0 h1"
      >
        <h1>
          Math Magicians
        </h1>
      </span>
      <div
        className="AllLinks"
      >
        <a
          className="Link border"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          className="Link border"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
        <a
          className="Link"
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </div>
    </div>
  </nav>
</div>
`;
